bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -0,0 +1,244 @@
+        <ul class="apps-marketplace">
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/facebook.png') }}" width="142" height="142" alt="Facebook" />
+            <span>Facebook</span>
+          </li>
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/youtube.png') }}" width="142" height="142" alt="YouTube" />
+            <span>YouTube</span>
+          </li>
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/line.png') }}" width="142" height="142" alt="Line" />
+            <span>Line</span>
+          </li>
+        </ul>
+        <a href="https://marketplace.firefox.com/" class="cta-button marketplace">{{_('Go to the Marketplace')}}</a>
+        <p class="footnote">{{_('* Pre-installed apps may vary by manufacturers and operators.')}}</p>
+      </div>
+    </section>
+
+    {% set feed_links = firefox_os_feed_links(LANG) %}
+    {% if feed_links %}
+    <section class="fxos-news">
+      <div class="content-wrapper">
+        <aside>
+          <h2>{{_('In the news')}}</h2>
+          {% set see_more = get_fxos_press_blog_link(LANG) %}
+          <p><a href="{{ see_more }}">{{_('See more')}}</a></p>
+        </aside>
+        <ul>
+          {% for link, title in feed_links %}
+          <li><a href="{{ link }}">{{ title }}</a></li>
+          {% endfor %}
+        </ul>
+      </div>
+    </section>
+    {% endif %}
+
+    <section class="fxos-community">
+      <div class="content-wrapper">
+        <img src="{{ static('img/firefox/os-new/mozilla_groupshot.png') }}" width="260" height="260" alt="{{_('Mozilla community photo')}}" />
+        <h2>{{_('Powered by the Mozilla community')}}</h2>
+        <p>{{_('Mozilla is a non-profit organization fueled by a passionate global community. Together, we’re able to create and innovate with local interests in mind, while making the Web better and more accessible for everyone everywhere.')}}</p>
+        <p><a href="https://www.mozilla.org/">{{_('Learn more about Mozilla')}}</a></p>
+      </div>
+    </section>
+    <section class="fxos-help">
+      <div class="content-wrapper">
+        <div class="help">
+          <h3>{{_('Need Help?')}}</h3>
+          <ul>
+            <li><a href="https://support.mozilla.org/products/firefox-os">{{_('Firefox OS Support')}}</a></li>
+            <li><a href="https://find.firefox.com/">{{_('Find My Device')}}</a></li>
+          </ul>
+        </div>
+        <div class="help">
+          <h3>{{_('For Developers')}}</h3>
+          <ul>
+            <li><a href="https://developer.mozilla.org">{{_('Mozilla Developer Network')}}</a></li>
+            <li><a href="https://webmaker.org/appmaker">{{_('Mozilla Appmaker')}}</a></li>
+            <li><a href="https://www.mozilla.org/firefox/os/devices/">{{_('Firefox OS devices')}}</a></li>
+            <li><a href="https://www.mozilla.org/firefox/os/devices/tv/">{{_('Other Firefox OS versions')}}</a></li>
```

Comment:
Those automatically get a name equal to the template name with `/` replaced with `.` and no file extension. So that's `{{ url('firefox.os.devices') }}` in your example.